docs/book/src/reference/envtest.md

```diff
@@ -104,9 +104,7 @@ var _ = BeforeSuite(func(done Done) {
 
 	_, err := testenv.Start()
 	Expect(err).NotTo(HaveOccurred())
-
-	close(done)
-}, 60)
```

Comment:
Why the done was removed here?

Reply:
https://onsi.github.io/ginkgo/MIGRATING_TO_V2#removed-async-testing

Reply:
Shows that the migration for this one is: 

```
It("...", func() {
	done := make(chan interface{})
	go func() {
		// user test code to run asynchronously
		close(done) //signifies the code is done
	}()
	Eventually(done, timeout).Should(BeClosed())
})
```

Reply:
yeah, if `async` is important for us, we can do that.
Also, custom reports~